protocol/contracts/lease/src/contract/state/opened/close/customer_close/full.rs

```diff
@@ -22,7 +22,8 @@ use crate::{
 };
 
 type Spec = FullClose;
-pub(super) type RepayableImpl = Close<Spec>;
+pub(in crate::contract::state::opened::close) type RepayableImpl = Close<Spec>;
```

Comment:
[multiple] I think this should work fine and would be a bit easier to read.
```suggestion
pub(in super::super) type RepayableImpl = Close<Spec>;
```

Reply:
there is no such thing as `super::super`. You cannot have anything after the first `super`.  The Rust compiler complains with a `Syntax error: expected an item`

Reply:
This works perfectly fine when the module `s` is not at the root level (lib.rs/main.rs). I'm guessing you missed the `in ` prefix.
![image](https://github.com/nolus-protocol/nolus-money-market/assets/80464733/2f33f9aa-8064-42b4-a3ea-12399f281bc6)

Reply:
thanks for the remark, that was the case